constrain/Classes/Constraints.swift

```diff
@@ -129,48 +108,10 @@ extension Constraints {
     
     fileprivate func finalizeConstraint(_ constraint: NSLayoutConstraint, _ identifier: ConstraintIdentifier) {
         view?.translatesAutoresizingMaskIntoConstraints = false
-        constraint.isActive = true
+        constraint.isActive = isActive
```

Comment:
FYI decided to make new constraints follow the toggle state of the saved ones for consistency, seemed more expected.